src/forms/inputs.js

```diff
@@ -19,15 +19,23 @@ import {
 *
 * @returns {object} React element
 */
-const BigButton = function ({ children, ...buttonProps }) {
+const BigButton = function ({ children, className, overrides }) {
```

Comment:
This actually seems to have broken the BigButton because some props, such as `onClick` are no longer passed to the underlying button. I'll open a new issue for it.

Reply:
I'm curious if you have other thoughts about how to handle this. I'm not a huge fan of passing on any random prop unfiltered, but `overrides` seems like a pain to maintain.